### DISCUSSION TITLE
error[E0599]: no function or associated item named `new` found for trait object `dyn RAND` in the current scope

### DISCUSSION THREAD

**Discussion by ttduongg** [lucasw/rust_test#24] [Q&A]
![image](https://user-images.githubusercontent.com/85562991/121190140-f3724780-c894-11eb-9c0e-5461133ef466.png)

**Comment by lucasw** [comment]
Are you trying to build something in rust_test- if so which branch?  It doesn't look like it from the directory you are running cargo in.
